mix.exs

```diff
@@ -2,26 +2,27 @@ defmodule NervesWpaSupplicant.Mixfile do
   use Mix.Project
 
   def project do
-    [app: :nerves_wpa_supplicant,
-     version: "0.3.2",
-     elixir: "~> 1.4",
-     build_embedded: Mix.env == :prod,
-     start_permanent: Mix.env == :prod,
-     compilers: [:elixir_make] ++ Mix.compilers,
-     make_clean: ["clean"],
-     deps: deps(),
+    [
+      app: :nerves_wpa_supplicant,
+      version: "0.3.2",
+      elixir: "~> 1.4",
+      build_embedded: Mix.env() == :prod,
+      start_permanent: Mix.env() == :prod,
+      compilers: [:elixir_make] ++ Mix.compilers(),
+      make_clean: ["clean"],
+      deps: deps(),
       docs: [extras: ["README.md"]],
-     package: package(),
-     description: description()
+      aliases: [format: [&format_c/1, "format"]],
```

Comment:
I guess that it's optional that you fix it then. I like having CI check the formatting so that I don't have `mix format` commits in the change history when I forget. I'm still getting used to some of the formatting rules and part of me is getting really lazy and writing code all over the place with the expectation that `mix format` fixes it. In any case, I'm good with your PR since updating the formating was long overdue.

Reply:
:+1: Will get more aggressive with CI and whatnot next pass.